### DISCUSSION TITLE
Helm charts around the operator

### DISCUSSION THREAD

**Discussion by arpan57** [hyperspike/valkey-operator#25] [Ideas]
Hi @dmolik, 

It would be great if we can have the helm charts around the operator. This would help installing the chart on the cluster or upgrade the operator versions, customization etc.  For example - https://github.com/OT-CONTAINER-KIT/redis-operator

Adding this as an idea. :-)

**Comment by dmolik** [comment]
I'm not quite focused on this ATM, but PRs are welcome!

**Comment by dmolik** [comment]
Landed up using helmify to generate a quick chart.

**Reply by arpan57** [reply]
Thank you.

**Comment by arpan57** [comment]
Hi @dmolik , 

I could start the operator pod on the k8s cluster. 
I notice that the valkey-cluster.yaml would spawn a valkey cluster, is it possible to create cluster chart? 
Something like https://github.com/OT-CONTAINER-KIT/redis-operator/tree/master/charts 

Thanks,
Arpan
